dashboard/test/lib/services/script_seed_test.rb

```diff
@@ -254,8 +254,8 @@ class ScriptSeedTest < ActiveSupport::TestCase
       assert_script_trees_equal script_with_changes, script
       lesson = script.lessons.first
       assert_equal(
-        ['Updated Activity Name', 'My Activity', 'New Activity Name'],
-        lesson.lesson_activities.map(&:name)
+        ["Updated Activity Name", "My Activity", "New Activity Name"],
+        lesson.lesson_activities.sort_by(&:position).map(&:name)
```

Comment:
I like the idea, but it looks like lesson_activities are already sorted by position, sadly: https://github.com/code-dot-org/code-dot-org/blob/7981d80e8bc7a84e7902658872c5adadc141e32f/dashboard/app/models/lesson.rb#L36

Reply:
I was just going to sort the names but that doesn't seem right because order matters here right?

Reply:
right, the order of the names should be preserved.